api-docs/internal/CioInternalCommon.api

```diff
@@ -69,6 +69,7 @@ public final class CioInternalCommon.CustomerIO {
 	public var registeredDeviceToken: String?;
 	public fun func resetSharedTestEnvironment();
 	public fun func initializeSharedInstance(with implementation: CustomerIOInstance);
+	public fun func setProfileAttributes(_ attributes: List<String: Any>);
```

Comment:
Shouldn't it show `Map` here instead of `List`, like below? Could be a bug in API generation snippet?

```
	public fun func setProfileAttributes(_ attributes: Map<String, Any>);
```